spec/support/factory_girl.rb

```diff
@@ -2,9 +2,10 @@
 
 FactoryGirl.define do
   factory :user do
-    sequence(:email) {|n| "user#{n}@example.com" }
-    password 'password'
-    password_confirmation 'password'
+    provider "github"
+    sequence (:uid) { |n| n }
+    sequence(:email) { |n| "jarlax{n}@launchacademy.com" }
```

Comment:
Unused block argument - `n`. You can omit the argument if you don't care about it.